scripts/find-lang.sh

```diff
@@ -54,7 +55,7 @@ fi
 shift
 
 if [ -z "$1" ] ; then usage
-else NAME=$1
+else NAMES[0]=$1
```

Comment:
Arrays are supported in POSIX shell, but not like that. They're way more rudimentary.